aws/resource_aws_efs_file_system_test.go

```diff
@@ -58,6 +57,29 @@ func TestAccAWSEFSFileSystem_importBasic(t *testing.T) {
 	})
 }
 
+func TestAccAWSEFSFileSystem_importWithLifecyclePolicy(t *testing.T) {
```

Comment:
Since the other new acceptance test includes import testing (🎉 ), this test is extraneous and can be removed. 👍

Reply:
other new acceptance test?  Is that something upstream I need to rebase onto my fork?  And then remove this entire `func TestAccAWSEFSFileSystem_importWithLifecyclePolicy(t *testing.T)` function and any refs to it?